dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -51,29 +50,29 @@ def organizer_enrollment_receipt(enrollment)
     @teacher_dashboard_url = CDO.code_org_url "/teacher-dashboard#/sections/#{@workshop.section_id}/manage"
 
     mail content_type: 'text/html',
+      from: from_no_reply,
       subject: 'Code.org workshop registration',
-      to: email_address(@workshop.organizer.name, @workshop.organizer.email),
-      reply_to: 'noreply@code.org'
+      to: email_address(@workshop.organizer.name, @workshop.organizer.email)
```

Comment:
No, this seems to be a github issue as a result of multiple merges and picking up other commits. If you look at the history for this file on staging after my merge, I don't appear to have made any changes. (https://github.com/code-dot-org/code-dot-org/commits/staging/dashboard/app/mailers/pd/workshop_mailer.rb)

And the "new" version on the right here matches the latest commits from staging, so it doesn't appear anything was changed by accident.